app/services/BulletPointBuilder.scala

```diff
@@ -102,28 +106,42 @@ object BulletPointBuilder {
   def locateTextsAndMatchesContainingLeadingText(
       noOfWordsToMatch: Int,
       inputTexts: List[String],
+      textsProcessed: Int,
       inputMatches: List[Match],
+      matchesProcessed: Int,
       outputTexts: Seq[String],
```

Comment:
There's a mixing of Lists and Seqs and subsequent toLists, would be better to stick with one or other and remove need for conversions

Reply:
Done